### PR TITLE
QuestionnaireResponse CSV Output

### PR DESCRIPTION
**Pull request by ivan-c**
* Adds `format` querystring parameter, allowing csv to be specified
  * Generates a CSV file with a row for every FHIR `answer`
  * Fields:
    * identifier (inline JSON)
    * patient_identifiers (inline JSON)
    * authored
    * instrument
    * question_code
    * answer_type
    * answer

See [PT#139422745](https://www.pivotaltracker.com/story/show/139422745)

### CODE REVIEW

**Comment by ivan-c** [comment]
Need to make [a few changes](https://www.pivotaltracker.com/story/show/139422745/comments/164412487), should be ready for review/merge afterwards

File: portal/models/fhir.py [hunk 1]
```diff
@@ -474,9 +474,146 @@ def __str__(self):
...
+                    break
+            else:
+                return identifier['value']
+        return None
+
+    def consolidate_answer_pairs(answers):
+        """
+        Merge paired answers (code and corresponding text) into single row/answer
+
+        Codes are the preferred way of referring to options but option text (at the time of administration) may be submitted alongside coded answers for ease of display
```
Comment (pbugni) [inline portal/models/fhir.py]: [Pep 8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) - please wrap by 79 chars

File: portal/views/assessment_engine.py [hunk 2]
```diff
@@ -602,11 +602,19 @@ def get_assessments():
     tags:
       - Assessment Engine
     parameters:
+      - name: format
+        in: query
+        description: format of file to download (CSV or JSON)
```
Comment (pbugni) [inline portal/views/assessment_engine.py]: Should probably point out the default if `format` isn't defined.

Comment (ivan-c) [inline portal/views/assessment_engine.py]: Should be fixed, not sure why Github isn't marking it as changed (maybe I added it too many lines below the referenced lines?)

**Pull request merged**